Model/Cc.php

```diff
@@ -495,9 +498,12 @@ protected function _buildCheckoutRequest($payment, $amount)
         }
         $request['Items']=[];
         foreach ($order->getAllVisibleItems() as $item) {
+            if ($item->getProductType() == "simple" && ($item->getParentItem())) {
```

Comment:
I think this is odd enough logic that a comment would help us remember why we are doing this for the future.  For example, "For variant items, the item gets added to the cart twice. Once without configuration details and one with configuration settings. This logic skips the duplicate item information without configuration details"

Also, this is a handy way for me to double check that how I'm reading this is accurate haha so please correct me if I'm wrong.

Reply:
Sure thing. I shall add the commit # once it is up in github. At present why the logic was used is explained only in the JIRA ticket.

Reply:
Commit 451f0032ee87472a3926180be34681c9f0f054e4
Thanks.